mono/mini/exceptions-x86.c

```diff
@@ -92,7 +92,7 @@ mono_win32_get_handle_stackoverflow (void)
 	x86_mov_reg_membase (code, X86_EBX, X86_ESP, 4, 4);
 
 	/* move current stack into edi for later restore */
-	x86_mov_reg_reg (code, X86_EDI, X86_ESP, 4);
+	x86_mov_reg_reg (code, X86_EDI, X86_ESP);
 
```

Comment:
This looks ok. The commit message says 'x86_mov_membase_reg' through.

Reply:
True. I was looking at that too. I'd fix the squash/commit-to-master message.
Please clarify:
1. I have convinced you of the overall PR and it is approved?
or 2. Just removing the usually/always==4 last parameter?